packages/overture-schema-addresses-theme/src/overture/schema/addresses/address/models.py

```diff
@@ -0,0 +1,80 @@
+"""Address feature models for Overture Maps addresses theme."""
```

Comment:
Not sure how much conviction I have in this, but I'm tempted to suggest that we re-use the following module pattern across themes:

1. `feature` - Contains the feature types.
2. `type` - Contains the other reusable types.

Reply:
The current version follows a pattern with 3 modules:

- `models.py` -- things that extend `BaseModel`. Only the primary one in a type module is re-exported from `__init__.py`; the rest can be imported from `.<theme>.<type>.models`
- `enums.py` -- enums (with no dependency on other modules)
- `types.py` -- type aliases. If a type alias incorporates a model, it goes into `models.py` to avoid circular imports.